generators/entity-client/templates/react/src/main/webapp/app/entities/entity.tsx.ejs

```diff
@@ -56,7 +56,7 @@ export interface I<%= entityReactName %>Props {
   <%_ if (searchEngine === 'elasticsearch') { _%>
   getSearchEntities: ICrudGetAllAction;
   <%_ } _%>
-  <%=entityInstancePlural %>: any[];
+  <%=entityInstancePlural %>All: any[];
```

Comment:
Can we change this to `<%=entityInstance %>List`, I think it's more clear what that variable holds.  Other than that looks good to me 👍

Reply:
Alright.